Cargo.lock

```diff
@@ -963,6 +964,16 @@ version = "0.2.11"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4cbbc9d0964165b47557570cce6c952866c2678457aca742aafc9fb771d30270"
 
+[[package]]
+name = "base256emoji"
```

Comment:
That is one whacky encoding scheme: https://github.com/multiformats/multibase/blob/master/rfcs/Base256Emoji.md